[2021-06-25 09:20:13,437] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2020-03-21T00:00:00+00:00 [queued]>
[2021-06-25 09:20:13,601] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2020-03-21T00:00:00+00:00 [queued]>
[2021-06-25 09:20:13,602] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:20:13,603] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-25 09:20:13,605] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:20:13,831] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2020-03-21T00:00:00+00:00
[2021-06-25 09:20:13,901] {standard_task_runner.py:52} INFO - Started process 9650 to run task
[2021-06-25 09:20:13,966] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2020-03-21T00:00:00+00:00', '--job-id', '348', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpiqm77_3z', '--error-file', '/tmp/tmpjts_i284']
[2021-06-25 09:20:13,992] {standard_task_runner.py:77} INFO - Job 348: Subtask task_4
[2021-06-25 09:20:14,442] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2020-03-21T00:00:00+00:00 [running]> on host airflowvm
[2021-06-25 09:20:14,995] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2020-03-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-21T00:00:00+00:00
[2021-06-25 09:20:14,998] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-25 09:20:15,001] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-25 09:20:15,034] {subprocess.py:75} INFO - Output:
[2021-06-25 09:20:18,071] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-25 09:20:18,330] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20200321T000000, start_date=20210625T092013, end_date=20210625T092018
[2021-06-25 09:20:18,706] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-25 09:20:18,784] {local_task_job.py:151} INFO - Task exited with return code 0
